vars/iqeUtils.groovy

```diff
@@ -517,24 +517,25 @@ def configIQE(String appName, Map options) {
     sh "rm -f \"${env.WORKSPACE}/.env\""
     writeEnv('ENV_FOR_DYNACONF', options['envName'])
 
-    if (options['useCustomSettings'] || options['settingsFromGit'] || options['settingsFileCredentialsId']) {
-        if (options['settingsFromGit']) {
-            getSettingsFromGit(
-                options['settingsGitRepo'],
-                options['settingsGitPath'],
-                options['settingsGitCredentialsId'],
-                options['settingsGitBranch'],
-                settingsDir,
-                appName
-            )
-        }
-        else if (options['settingsFileCredentialsId']) {
-            getSettingsFromJenkinsSecret(options['settingsFileCredentialsId'], settingsDir)
-        }
-        writeEnv("IQE_TESTS_${appName.toUpperCase()}_CONF_PATH", settingsDir)
+    if (options['settingsFromGit']) {
+        getSettingsFromGit(
+            options['settingsGitRepo'],
+            options['settingsGitPath'],
+            options['settingsGitCredentialsId'],
+            options['settingsGitBranch'],
+            settingsDir,
+            appName
+        )
+    }
+    else if (options['settingsFileCredentialsId']) {
```

Comment:
According to the email, I understand that the preference will be to set `settingsFromGit` to False. That will mean that we fall into this branch of the if, because `settingsFileCredentialsId` is set to a default value in https://github.com/RedHatInsights/insights-pipeline-lib/blob/master/vars/iqeUtils.groovy#L97 And if that file does not exit, then the pipeline will fail.

Reply:
This section of code is actually how it looked before #185 and #187 -- so this is behaviour that was already there.

Reply:
Since this is an 'else if' -- if settingsFromGit is true, we won't hit this block

Reply:
yes, my point is that `settingsFileCredentialsId` has some default value that needs to be explicitly nullify (e.g. passing `settingsFileCredentialsId = ''`). But to be honest I do not know what this value represent and where it should/can be placed the corresponding file.

Reply:
It refers to the name of a secret stored in Jenkins. At this point it's now a legacy thing that was used before we started storing settings in conf files in github. We used to store .yaml config files as secrets in Jenkins.

Reply:
Thanks for the explanation